shared/src/commonMain/kotlin/io/customer/shared/sdk/meta/Workspace.kt

```diff
@@ -46,7 +46,7 @@ data class Workspace constructor(
      * auto initializing the SDK.
      */
     internal object DefaultValue {
-        val IDENTITY_TYPE = IdentityType.AUTO_IDENTIFY
+        val IDENTITY_TYPE = IdentityType.ID
```

Comment:
This is temporary, we'll probably be making it mandatory to specify the identity type when initializing the SDK.